gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/MergeDiagnosticsJsonTask.java

```diff
@@ -16,7 +16,4 @@
 
 package com.palantir.gradle.dist.service;
 
-import org.gradle.api.tasks.CacheableTask;
-
-@CacheableTask
 public abstract class MergeDiagnosticsJsonTask extends MergeDiagnosticsJsonTaskImpl {}
```

Comment:
Might be worth a comment so we don't attempt to re-"optimize" this in the future!

Reply:
hmm, I think in that case we would end up putting that comment on every single simple task we write...

Reply:
I don't think it would be entirely unreasonable to explicitly declare whether each task is cacheable or not, but I defer to your judgement.